PyInstaller/depend/bindepend.py

```diff
@@ -873,3 +873,19 @@ def get_python_library_path():
 
     # Python library NOT found. Return just None.
     return None
+
+def findSystemLibrary(name):
+    '''
+        Given a library name, try to resolve the path to that library. If the
+        path is already an absolute path, return that without searching.
+    '''
+
+    if os.path.isabs(name):
+        return name
+
+    if is_unix:
+        return findLibrary(name)
+    elif is_win:
+        return getfullnameof(name)
+    else:
+        raise ValueError("findSystemLibrary not implemented for this platform")
```

Comment:
Okay, so we leave it this way for now. Thanks for taking the time analyzing this.